packages/turf-polygon-smooth/test.js

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs';
+import test from 'tape';
+import path from 'path';
+import load from 'load-json-file';
+import write from 'write-json-file';
+import polygonSmooth from '.';
+
+const directories = {
+    in: path.join(__dirname, 'test', 'in') + path.sep,
+    out: path.join(__dirname, 'test', 'out') + path.sep
+};
+
+const fixtures = fs.readdirSync(directories.in).map(filename => {
+    return {
+        filename,
+        name: path.parse(filename).name,
+        geojson: load.sync(directories.in + filename)
+    };
+});
+process.env.REGEN = true
```

Comment:
Having a hard time define environment variables on Windows 😉

Reply:
yes - stupid stinkin environment variables, always forget how to set them because it has to be so super specific!

Reply:
True that, but once the fixtures are defined, "in theory"  they shouldn't be changing.

I think in Windows you can use:

```
setx DEBUG true
```

https://technet.microsoft.com/en-us/library/cc755104(v=ws.11).aspx